tracer/src/Datadog.Trace/Logging/Internal/DatadogRollingFileSink.cs

```diff
@@ -0,0 +1,249 @@
+// <copyright file="DatadogRollingFileSink.cs" company="Datadog">
```

Comment:
This is file is basically identical to `RollingFileSink` right, just it creates a `DatadogDeferredSink` instead of File/Shared Sink? Maybe we should add a comment about that?